modules/cluster-logging-rn-5.3.9.adoc

```diff
@@ -0,0 +1,32 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-9"]
+= Logging 5.3.9
+This release includes link:https://access.redhat.com/errata/RHBA-2022:5557[RHBA-2022:5557-OpenShift Logging Bug Fix Release 5.3.9].
+
+[id="openshift-logging-5-3-9-bug-fixes"]
+== Bug fixes
+
+* Before this update, the logging collector would include path as a label for the metrics it produced. This path changes frequently and contributed to significant storage changes for the Prometheus server. With this update, the label has been dropped to resolve the issue and reduce storage consumption.(link:https://issues.redhat.com/browse/LOG-2682[LOG-2682])
```

Comment:
```suggestion
* Before this update, the logging collector included a path as a label for the metrics it produced. This path changed frequently and contributed to significant storage changes for the Prometheus server. With this update, the label has been dropped to resolve the issue and reduce storage consumption.(link:https://issues.redhat.com/browse/LOG-2682[LOG-2682])
```
Sorry I didn't see this yesterday. Added an article before `path` and removed `would`:
IBM SG: Use [would] only to refer to an unfulfilled condition or hypothesis and to express the future in the past, for example, "The command
was rejected because it would have resulted in the ClusterName attribute being blank."